### ISSUE TITLE
Generated code wraps update sql data with parenthesis

### ISSUE THREAD

**Issue by ProfChaos** [stephenafamo/bob#146]
After updating to v0.23.0 the generated code produces the error below in sql. I tried downgrading to v0.22.0 and everything works as expected.

This is a postgres database and we are not using any custom templates for bob. 

```
postgres-1  | 2024-01-04 07:34:21.716 UTC [2062] ERROR:  syntax error at or near "=" at character 47
postgres-1  | 2024-01-04 07:34:21.716 UTC [2062] STATEMENT:  UPDATE "blocks" AS "blocks" SET
postgres-1  | 	("deleted_at" = $1)
postgres-1  | 	WHERE ("id" IN ($2))
```

```
postgres-1  | 2024-01-04 07:34:16.685 UTC [2050] ERROR:  syntax error at or near "=" at character 47
postgres-1  | 2024-01-04 07:34:16.685 UTC [2050] STATEMENT:  UPDATE "blocks" AS "blocks" SET
postgres-1  | 	("updated_at" = $1),
postgres-1  | 	("deleted_at" = $2)
postgres-1  | 	WHERE ("id" IN ($3))
```

It wraps the data to be updated in parenthesis

```golang
blocks, err := models.Blocks.Query(...).All()  
err = blocks.UpdateAll(ctx, storage.Bob, models.BlockSetter{DeletedAt: omitnull.From(time.Now())})
```

The same goes for the `(*block).Update(...)`

**Comment by stephenafamo** [comment]
Fixed in `v0.23.1` 

Although `v0.23.2` has also been released with additional fixes.

**Comment by ProfChaos** [comment]
Awesome, thanks!
